src/sparql/icd11foundation-relevant-signature.sparql

```diff
@@ -15,4 +14,7 @@ WHERE {
     }
   }
   FILTER(isIRI(?term))
+  FILTER NOT EXISTS {
+    ?term rdfs:subClassOf* <http://id.who.int/icd/entity/979408586> .
```

Comment:
## Running build
Currently I'm running the build, for the workflow:
`main` <-- this PR <-- build PR

When the build is done I'll open a PR for it and link that here.

Reply:
I don't follow the PR path here. The eventual endpoint here should be a new build run in `main` that has the ICD11 Foundation content mapped from content created with this new SPARQL query as well as any other new files pulled from the `mondo` repo, e.g. to account for the latest curation Sabrina did for DO

Reply:
Yep, that will be the result! I created this branch off of `main`, so this workflow will have the exact same results, and I could be wrong but I think it is more in-line with the SOP. The workflow has the benefit of not causing any issues on `main` in the event of something being wrong or unexpected with the changes.

Reply:
Build done!
- #549

However:
_Trish wrote:_
> Let's also make sure there is a `unmapped_icd11foundation_lex_exact.tsv` file in the results from the pipeline since I do not see that file in the build I ran last week.

I'm not seeing that file in the outputs ;(. Do you want to take a preliminary look into why this may be the case? I won't be able to look at this until tomorrow afternoon, but I can look into it then.

Reply:
The `unmapped_icd11foundation_lex_exact.tsv` is in this build.

Reply:
Oh, so it is. It was late and I was heading to bed, and I ls'ed the `unmapped/` directory instead of the `lexmatch/` lol.